src/main-client.ts

```diff
@@ -1177,2206 +657,63 @@ export class MainClient extends BaseRestClient {
 
   /**
    *
-   * Market Data Endpoints
+   * User Data Stream Endpoints
    *
    **/
 
-  testConnectivity(): Promise<{}> {
-    return this.get('api/v3/ping');
-  }
-
-  getExchangeInfo(params?: ExchangeInfoParams): Promise<ExchangeInfo> {
-    const symbols = params?.symbols && JSON.stringify(params.symbols);
-    const symbol = params?.symbol;
-
-    let urlSuffix = '';
-    if (symbol) {
-      urlSuffix += '?symbol=' + symbol;
-    } else if (symbols) {
-      urlSuffix += '?symbols=' + symbols;
-    }
-
-    return this.get('api/v3/exchangeInfo' + urlSuffix);
-  }
-
-  getOrderBook(params: OrderBookParams): Promise<OrderBookResponse> {
-    return this.get('api/v3/depth', params);
+  // spot
+  getSpotUserDataListenKey(): Promise<{ listenKey: string }> {
```

Comment:
You can include all of these listen key endpoints as one "category" in the v2 main client. Same for futures and the futures v2 client. Easier to see if they're all grouped together

Reply:
Included.